### DISCUSSION TITLE
Cannot Remove/Reinstall/Format/Erase Ventoy from the USB

### DISCUSSION THREAD

**Discussion by bariserkus** [ventoy/Ventoy#1474] [Q&A]
I have tried many things to remove/format the USB that I installed Ventoy but I could not. I also used Ventoy tool itself, to erase, reinstall again but it did not work. Please help me to salvage my USB. I cannot use it with Windows. I can see the USB drive mounted on Linux but cannot use it properly. What is the low-level method to format this USB and completely get rid of Ventoy?

**Comment by steve6375** [comment]
'it did not work'
Can you give any details? Did you see any error messages? Did you try a partition tool? 
https://linuxhint.com/delete-a-partition-in-linux/

Note that cheap, fake USB drives often go faulty and become write-protected.

**Reply by bariserkus** [reply]
Thanks for the reply. I have tried the link you gave. The USB is Sandisk Ultra USB 3.0 64 GB. I have also tried gpart, gdisk, but I have encountered errors: such as I need to reboot for the new table to be used, or the drive is being used. There should be a more straightforward way of removing the Ventoy. If your suggested way should already clear ventoy from the USB, then, maybe the USB gone bad?

**Reply by steve6375** [reply]
You need to boot from your hard disk. Windows OS or Linux OS?
Or Are you booting from the USB drive itself to Ventoy and then trying to erase the contents? This may not work under Linux.

If booting on a Windows system try RMPrepUSB.exe - CLEAN button, then safely remove and unplug - replug.

**Reply by bariserkus** [reply]
I am booting from my Linux. I have tried to erase it from my Windows (not booted from Ventoy USB), but in Windows Ventory USB freezes when I want to access with Windows Disk Management Tool, Rufus or similar USB tools.

**Reply by bariserkus** [reply]
I have tried RMPrepUSB.exe as well, but as I mentioned, the program freezes as soon as I inserted USB. In fact windows explorer also freezes. So the USB is probably completely ruined. ?

**Comment by HerbJul** [comment]
Same Problem her with Kingston USB Traveler

**Reply by nike1v** [reply]
Wow, i just faced the same issue with the same usb druve. Mune one is 128gb, i dont really want it to be stuck on read only mode, but 12 hours or constant trials give no result. Do you have any ideas?

**Reply by LupusEtFlos** [reply]
Just had the same happen to a 64GB DataTraveler Exodia M... it was my boss' USB stick, he saw my generic 32GB stick with Ventoy working fine and went nuts with it, now what do I do? :/

**Comment by steve6375** [comment]
https://rmprepusb.com/tutorials/007-all-about-fake-sd-cards-and-usb-flash-drives/
https://rmprepusb.com/tutorials/054-how-to-fix-write-protected-read-only-drives-and-sd-cards/

Ventoy has had millions of downloads and is used by millions of people without any problem.
There are also millions of cheap fake flash drives around.
Unlike when in normal use, Ventoy creates a partition at the very end of the USB drive and writes to these sectors. So Ventoy will often be seen to 'destroy' a USB flash drive but this is only because the flash drive has bad firmware and is a fake in the first place. If you had filled up that flash drive to the end, you would also have destroyed it.
